### PR TITLE
Kupper

### PR DESCRIPTION
**Pull request by kupperes**
Added login page, signup page, reformatted base page, added some base css stuff.

### CODE REVIEW

File: secret_santa/views.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
-from django.shortcuts import render
+from django.contrib.auth.forms import UserCreationForm
+from django.urls import reverse_lazy
+from django.views import generic
```
Comment (baierpa) [inline secret_santa/views.py]: what do these imports do?

**Comment by baierpa** [comment]
getting a `TemplateDoesNotExist: base.html` error? where does that file exist?

File: secret_santa/views.py (hunk 1 above)
Comment (kupperes) [reply secret_santa/views.py]: These are left over from the previous iteration.  All of the imports are also [correctly] in the accounts/views.py.  I originally deleted this views file, but it didn't like that so I put it back. 

-django.contrib.auth.forms imports a django supplied form used to create the user object which stores username and password.

-django.urls import reverse_lazy is a function of sorts that allow a redirect back to whatever page you originally came from.  This is used primarily when logging in so when that is complete you go back to where you started...in this case the homepage.  

-django.views import generic is the stock django logic used to add users to the database

**Comment by kupperes** [comment]
I made a new push with two a few missing documents.  Everything should be working now.

**Pull request merged**